packages/common/src/services/pagination.service.ts

```diff
@@ -13,6 +14,7 @@ import type {
 import type { BackendUtilityService } from './backendUtility.service';
 import type { SharedService } from './shared.service';
 import type { Observable, RxJsFacade } from './rxjsFacade';
+import { PageInfo } from '../interfaces/pageInfo.interface';
```

Comment:
same as earlier, please add `type` to the import

Reply:
I've deleted that line and made the import  from the index file